### PR TITLE
Make password hint configurable from settings/defaults.ini

### LINKED ISSUE #15867: Ability to change the password hint like login hint
**What would you like to be added**:
Ability to change the password hint like login hint

**Why is this needed**:
Would like the password to be editable like login hint is. Do not want to modify the template, for upgrade reasons.

### PR DESCRIPTION
**Pull request by joshuawoodward**
**What this PR does / why we need it**:
Make password hint configurable from settings/defaults.ini

**Which issue(s) this PR fixes**:
Fixes #15867

### CODE REVIEW

File: conf/defaults.ini [hunk 1]
```diff
@@ -231,6 +231,7 @@ verify_email_enabled = false
 
 # Background text for the user field on the login page
 login_hint = email or username
+passowrd_hint = password
```
Comment (bergquist) [inline conf/defaults.ini]: this should be added to `sample.ini` and `docs/configuration.md` as well with a comment explaining what it does.

**Review by bergquist — CHANGES_REQUESTED** [review]
just a minor change required and we will merge this.

File: conf/defaults.ini (hunk 1 above)
Comment (torkelo) [reply conf/defaults.ini]: and spelled correctly? :)

**Comment by torkelo** [comment]
fixed spelling added to sample.ini & configuraiton docs & merged

**Pull request merged**